Exercise-master/src/main/resources/application.properties

```diff
@@ -1,7 +1,7 @@
 server.port=8081
-spring.datasource.url= jdbc:postgresql://localhost:5432/exercise
+spring.datasource.url= jdbc:postgresql://localhost:5432/quora
 spring.datasource.username=postgres
-spring.datasource.password=123
+spring.datasource.password=12345
```

Comment:
Not sure if we need to change database details in main code as this will be checked by upgrad and they might be using these settings.So these changes we can keep in our local and not check in